action.yml

```diff
@@ -43,7 +53,7 @@ runs:
     run: git fetch --prune --unshallow --tags
     shell: bash
   - name: Install upgrade-provider
-    run: go install github.com/pulumi/upgrade-provider@main
+    run: go install github.com/pulumi/upgrade-provider@1123b4b82f87da12510454ee2dd4330b5c92c958
```

Comment:
Never this back to main once https://github.com/pulumi/upgrade-provider/pull/176 merges